server/bundler/loader.js

```diff
@@ -165,6 +165,18 @@ function singleEnsure( chunkName ) {
 	return result.join( '\n' );
 }
 
+function fillSectionCSSUrls( sections ) {
+	sections.forEach( function( section ) {
+		if ( section.css ) {
+			section.cssUrls = {
+				ltr: utils.getHashedUrl( 'sections/' + section.css + '.css' ),
+				rtl: utils.getHashedUrl( 'sections-rtl/' + section.css + '.rtl.css' ),
+			};
+		}
+	} );
+	return sections;
```

Comment:
Why do we need to return the list of sections if it's not used when `fillSectionCSSUrls()` is called?